cql3/statements/alter_keyspace_statement.cc

```diff
@@ -43,18 +44,17 @@ future<> cql3::statements::alter_keyspace_statement::check_access(query_processo
     return state.has_keyspace_access(_name, auth::permission::ALTER);
 }
 
-static bool validate_rf_difference(const std::string_view curr_rf, const std::string_view new_rf) {
-    auto to_number = [] (const std::string_view rf) {
-        int result;
-        // We assume the passed string view represents a valid decimal number,
-        // so we don't need the error code.
-        (void) std::from_chars(rf.begin(), rf.end(), result);
-        return result;
-    };
-
-    // We want to ensure that each DC's RF is going to change by at most 1
-    // because in that case the old and new quorums must overlap.
-    return std::abs(to_number(curr_rf) - to_number(new_rf)) <= 1;
+// We want to ensure that each DC's RF is going to change by at most 1
+static bool validate_rf_difference(const std::string& curr_rf, const std::string& new_rf) {
```

Comment:
Why change `string_view` to `std::string&`?

Reply:
It's because I get this error:
```
cql3/statements/alter_keyspace_statement.cc:49:25: error: no matching function for call to 'stoi'
   49 |         return std::abs(std::stoi(curr_rf) - std::stoi(new_rf));
      |                         ^~~~~~~~~